### PR TITLE
Improve User Experience

### PR DESCRIPTION
**Pull request by dandrei279**
Add more notes in the UI
Make existing notes clearer

### CODE REVIEW

File: lang/en/qtype_topictagged.php [hunk 1]
```diff
@@ -28,10 +28,21 @@
 $string['pluginname_link'] = 'question/type/topictagged';
 $string['pluginnameadding'] = 'Adding a topic tagged question';
 $string['pluginnameediting'] = 'Editing a topictagged question';
-$string['pluginnamesummary'] = 'Select a question from the question bank having the selected difficulty and topic that has not been used in a while';
+$string['pluginnamesummary'] = 'Select a question from the question bank having the selected difficulty and' .
+    'topic that has not been used in a while.<br>' .
+    '<p><font color=#F57542><b>Warning:</b></font><br>' .
+    '<font color=#A14A28>Do not add this question into Question bank. Just add it to a quiz.</font></p><br>' .
+    '<p><font color=#4287F5><b>Note: </b></font>' .
+    '<font color=#1C3C70>Keep in mind that preview shows randomly questions selected by the selected difficulty and topic.</font></p>' .
```
Comment (StefanJum) [inline lang/en/qtype_topictagged.php:36]:
```suggestion
    '<font color=#1C3C70>Keep in mind that preview shows random questions selected by the given difficulty and topic, without updating the last_used tag.</font></p>' .
```

File: lang/en/qtype_topictagged.php [hunk 2]
```diff
@@ -28,10 +28,21 @@
 $string['pluginname_link'] = 'question/type/topictagged';
 $string['pluginnameadding'] = 'Adding a topic tagged question';
 $string['pluginnameediting'] = 'Editing a topictagged question';
-$string['pluginnamesummary'] = 'Select a question from the question bank having the selected difficulty and topic that has not been used in a while';
+$string['pluginnamesummary'] = 'Select a question from the question bank having the selected difficulty and' .
+    'topic that has not been used in a while.<br>' .
+    '<p><font color=#F57542><b>Warning:</b></font><br>' .
+    '<font color=#A14A28>Do not add this question into Question bank. Just add it to a quiz.</font></p><br>' .
```
Comment (StefanJum) [inline lang/en/qtype_topictagged.php:34]:
```suggestion
    '<font color=#A14A28>Do not add this question into the question bank. Just add it to a quiz.</font></p><br>' .
```

File: lang/en/qtype_topictagged.php [hunk 3]
```diff
@@ -66,5 +77,5 @@
...
 $string['download'] = 'Exporting Questions';
-$string['download_help'] = 'For exporting, choose the category you want to download the questions from and then choose the file format.<ol><li><b><i><a href=https://docs.moodle.org/310/en/Moodle_XML_format> MXML </a> (Recommended)</i></b> can be easily re-imported.</li><li><i>CSV (Complex)</i> easier to manipulate data as long as you have a local copy.<br>See <a href=https://github.com/systems-cs-pub-ro/quiz-manager-moodle/tree/master#exporting-questions> Exporting Questions </a> for more information.</li></ol>';
+$string['download_help'] = 'For exporting, choose the category you want to download the questions from and then choose the file format.<ol><li><b><i><a href=https://docs.moodle.org/310/en/Moodle_XML_format> MXML </a> (Recommended)</i></b> can be easily re-imported.</li><li><i>CSV (Complex)</i> easier to manipulate data as long as you have a local copy.<br>See <a href=https://github.com/systems-cs-pub-ro/quiz-manager-moodle#readme> Documentation </a> for more information.</li></ol>';
```
Comment (StefanJum) [inline lang/en/qtype_topictagged.php:80]:
```suggestion
$string['download_help'] = 'For exporting, choose the category you want to download the questions from and then choose the file format.<ol><li><b><i><a href=https://docs.moodle.org/310/en/Moodle_XML_format> MXML </a> (Recommended)</i></b> can be easily re-imported.</li><li><i>CSV (Complex)</i> easier to manipulate data as long as you have a local copy.<br>The CSV file will only contain questions used in a quiz.<br>See <a href=https://github.com/systems-cs-pub-ro/quiz-manager-moodle#readme> Documentation </a> for more information.</li></ol>';
```

**Review by StefanJum — APPROVED** [review]
Looks good!

File: lang/en/qtype_topictagged.php (hunk 1 above)
Comment (dandrei279) [reply lang/en/qtype_topictagged.php:36]: Do you think the final user (evaluator) should know about our internal representation?
What do you think about:
```suggestion
    '<font color=#1C3C70>Keep in mind that preview shows random questions selected by the given difficulty and topic, without affecting the selection process.</font></p>' .
```

Comment (StefanJum) [reply lang/en/qtype_topictagged.php:36]: You're right, the user should not be given more technical information than needed. This looks much better.

**Pull request merged**